app/views/catalog/_document_list.html.erb

```diff
@@ -0,0 +1,8 @@
+<% # Override to remove id="documents" on div. When documents are grouped it causes duplicate id issues %>
+<% # https://github.com/projectblacklight/blacklight/blob/v8.1.0/app/views/catalog/_document_list.html.erb %>
+<% # container for all documents in index list view -%>
+<% view_config = local_assigns[:view_config] || blacklight_config&.view_config(document_index_view_type) %>
+<div class="al-document-listings documents-<%= view_config&.key || document_index_view_type %>">
```

Comment:
After looking closer, I think we'll need to keep the `documents` id in a non-grouped view of search results (to accommodate the `Skip to First Result` skip-link in those views). Here's a recommended way to only remove it only when the results are grouped...
```
<div id="<%= 'documents' unless grouped? %>" class="...">
```

Reply:
Updated. I modified it slightly `<div <%= 'id="documents"'.html_safe unless grouped? %>` to avoid having empty ids for grouped records, `id=""`